mlrun/__main__.py

```diff
@@ -1006,11 +1007,18 @@ def logs(uid, project, offset, db, watch):
 @click.option(
     "--env-file", default="", help="path to .env file to load config/variables from"
 )
+# TODO: Remove --ensure-project in 1.6.0
 @click.option(
     "--ensure-project",
     is_flag=True,
     help="ensure the project exists, if not, create project",
 )
+@click.option(
+    "--save",
+    is_flag=True,
+    default=True,
+    help="create and save the project if not exist",
+)
```

Comment:
if default is True (when not passing) and this is a flag, so also passing it `--save` will be True, so I think you'll never reach False.

Reply:
take a look at - https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags 
guess we can allow both `save and no-save` in the same argument with default true